store/src/java/com/zimbra/cs/imap/ImapRequest.java

```diff
@@ -610,6 +614,51 @@ String readSequence() throws ImapParseException {
         return validateSequence(readContent(SEQUENCE_CHARS), true);
     }
 
+    CacheEntryType[] readCacheEntryTypes() throws IOException, ImapParseException {
+        String[] cacheStrTypes = readAstring().split(",");
```

Comment:
This is a good idea. I originally made it a CSV to match the format used by FlushCacheRequest, but this makes more sense.